subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/plugins/JavaGradlePluginPluginIntegrationTest.groovy

```diff
@@ -205,6 +205,26 @@ class JavaGradlePluginPluginIntegrationTest extends WellBehavedPluginTest {
         succeeds "jar"
     }
 
+    def "Generated plugin descriptor are os independent"() {
+        given:
+        buildFile()
+        goodPlugin()
+        buildFile << """
+            gradlePlugin {
+                plugins {
+                    testPlugin {
+                        id = 'test-plugin'
+                        implementationClass = 'com.xxx.TestPlugin'
+                    }
+                }
+            }
+        """
+        expect:
+        succeeds "jar"
+        def descriptorHash = file("build", "resources", "main", "META-INF", "gradle-plugins", "test-plugin.properties").md5Hash
```

Comment:
Why not use `file("build/resources/main/...")`?

Reply:
Done.